src/script/E2EIdentity/E2EIdentityEnrollment.ts

```diff
@@ -345,6 +345,7 @@ export class E2EIHandler extends TypedEventEmitter<Events> {
       secondaryActionFn: () => {
         this.showE2EINotificationMessage();
       },
+      hideSecondary: !!this.config?.isFreshMLSSelfClient,
```

Comment:
This is not enough actually.

This button needs also be hidden for renewals, after the end of snooze period

Reply:
I haven't added that check yet, you can add it as part of this PR

Reply:
Quick reminder that we have 5 cases for soft-lock

1. New Device without certificate, we ignore the snooze period
2. Existing Device without certificate, we soft lock after expired snooze period
3. Existing Device with certificate, we soft lock after expired snooze period for refresh
4. Existing Device with expired certificate, soft lock immediately
4. Revoked Device (not yet implemented)